File: Java/EmailSwap.java
import java.util.Scanner;
import java.util.regex.Pattern;
import java.util.regex.Matcher;

public class EmailValidator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an email address: ");
        String email = scanner.nextLine();

        // Regular expression for email validation
        String emailPattern = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$";
        Pattern pattern = Pattern.compile(emailPattern);
        Matcher matcher = pattern.matcher(email);

        if (matcher.matches()) {
            // Convert the email to lowercase
            email = email.toLowerCase();
            System.out.println("Valid Email (Lowercase): " + email);
        } else {
            System.out.println("Invalid Email");
        }

        scanner.close();
    }
}

File: Kotlin/EmailSwap.kt
import java.util.regex.Pattern

fun main() {
    print("Enter an email address: ")
    val email = readLine()

    // Regular expression for email validation
    val emailPattern = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
    val regex = Pattern.compile(emailPattern)

    if (email != null && regex.matcher(email).matches()) {
        // Convert the email to lowercase
        val lowercaseEmail = email.toLowerCase()
        println("Valid Email (Lowercase): $lowercaseEmail")
    } else {
        println("Invalid Email")
    }
}
